README.md

```diff
@@ -5,6 +5,21 @@ Demonstrates Iteria React Lowcode features:
 * [Translation sheet](https://www.youtube.com/watch?v=5-U2vfgHkMA&t=396s) - spreadsheet with all messages and their translations
 
 You can run this demo locally, [edit in StackBlitz](https://stackblitz.com/github/iteria-app/example-material-ui) or using Gitpod. 
+
 [![Open in Gitpod](https://gitpod.io/button/open-in-gitpod.svg)](https://gitpod.io/#https://github.com/iteria-app/example-material-ui)
 
 [![Demo Video](https://img.youtube.com/vi/5-U2vfgHkMA/0.jpg)](https://www.youtube.com/watch?v=5-U2vfgHkMA&t=206s)
+
+
+## Getting started
+
+Use this deploy button to get your own copy of the repository.
+
+[![Deploy to Netlify](https://www.netlify.com/img/deploy/button.svg)](https://app.netlify.com/start/deploy?repository=https://github.com/iteria-app/example-material-ui&stack=cms)
+
+This button will do the following:
+
+- Connect to your GitHub account and create a new repository with the name you specify
+- Deploy your copy of the repository and setup continuous deployment
+
+Now you're all set, and you can start customizing your static site!
```

Comment:
Now you're all set, and you can start customizing your static/jamstack site connected via GraphQL to Hasura!